onnx/hub.py

```diff
@@ -122,22 +121,38 @@ def _get_base_url(repo: str, lfs: bool = False) -> str:
         return "https://raw.githubusercontent.com/{}/{}/{}/".format(repo_owner, repo_name, repo_ref)
 
 
+def _download_file(url: str, file_name: str) -> None:
+    """
+    Downloads the file with specifed file_name from the url
+    @param url: a url of download link
```

Comment:
`:param`

Reply:
Good point. `:param` might be more prevalent than `@param`. Have just updated it globally in the same file. Thanks!